src/element/textWysiwyg.tsx

```diff
@@ -220,6 +222,9 @@ export const textWysiwyg = ({
       if (isTestEnv()) {
         editable.style.fontFamily = getFontFamilyString(updatedElement);
       }
+      if (isCollaborating) {
+        mutateElement(updatedElement, { x: coordX, y: coordY });
```

Comment:
isn't this the analogue for what we're doing on submit if not collaborating? If so, we're computing much more (and differently) on submit?

Reply:
Also, if this should work when collaborating (and it presumably doesn't affect perf), then why not do it all the time and reduce complexity (having to keep two separate implems)?

Reply:
Yes this things are same what we do on submit but the reason I am not doing it all time and specifically when `collab/submit` is I am worried about perf since `wrapText` is called on every change and mutation for coords will also happen on every change so If it affects perf in any way(Testing looks fine though) hence avoiding rn.

Reply:
I think it's a good idea to align the implems for 2 reasons: 

1. Complexity
2. Generally, perf is worse when collaborating. And it's much more likely we'll notice perf issues locally, thus more likely we will do something about it, rather than if it's hidden under collab.

Reply:
Also, in E+ we're collaborating pretty much all the time, anyway.

Reply:
Hmm makes sense, I have aligned the implementation to be same irrespective of collab/submit